File: week6/day2/switch-up.js
console.log('up & down')

var switchup = function(string) {

    var updown = "";

    for(var i = 0; i < string.length; i++){

        var letter = string[i];

        if (letter === letter.toUpperCase()){
            updown += letter.toLowerCase();
        } else {
            updown += letter.toUpperCase();
        }           
    }
    return updown;
}

console.log(switchup("Switch caSe FoR eVery letter"));
console.log(switchup("js_Was_created_bY BRENdan Eich"));

File: week06/day2/switch-up.rb

# string = "Switch caSe FoR eVery letter"
# p string.swapcase  

def switchup string

    converted = []

    string.each_char do |letter|
        if letter =~ /[A-Z]/
            up = letter.downcase
            converted.push up
        else
            low = letter.upcase
            converted.push low
        end
    end
    p converted.join('')
end

switchup "Switch caSe FoR eVery letter"
switchup "js_Was_created_bY BRENdan Eich"